packages/fluentui/react-northstar/src/components/Carousel/Carousel.tsx

```diff
@@ -170,15 +168,10 @@ class Carousel extends AutoControlledComponent<WithAsProp<CarouselProps>, Carous
 
   static getAutoControlledStateFromProps(props: CarouselProps, state: CarouselState) {
```

Comment:
as far as I can see, this was the only use for getAutoControlledStateFromProps and now it can probably be removed totally.